bot.py

```diff
@@ -553,7 +565,7 @@ async def inspire(ctx):
 
 
 @bot.command(name="color")
-async def color(ctx, *, hex):
+async def color(ctx, *, hex: any = commands.parameter(description="A hex color code")):
```

Comment:
not necessarily a hex code, see https://pillow.readthedocs.io/en/stable/reference/ImageColor.html